internal/tiltfile/skylarkTypes.go

```diff
@@ -112,9 +114,9 @@ func (*dockerImage) Hash() (uint32, error) {
 func (d *dockerImage) Attr(name string) (skylark.Value, error) {
 	switch name {
 	case "file_name":
-		return d.fileName, nil
+		return skylark.String(d.fileName), nil
```

Comment:
uhh @landism / @nicks am i doing this right?

Reply:
:+1: